src/components/popover-container/documentation/info.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
No need for this file, should all be in MDX.

Reply:
removed